### DISCUSSION TITLE
Tuya Local-proposing various devices smart power meter

### DISCUSSION THREAD

**Discussion by jnasserifar** [make-all/tuya-local#1106] [General]
I have two smart plug Atroch SWP1 and I tried the integration.
in my first try, the first plug proposed with Generic power meter (Energy monitoring smart plug) which is in the list of supported devices. I reviewed the id<s and they are matched with iot.tuya device properties.
It detected 5 entities for this device that I think its logical.

In my second try to add second plug the integration proposed 5 different supported devices, surprisingly Energy monitoring smart plug was NOT in the list!

I selected the clamp one
![image](https://github.com/make-all/tuya-local/assets/29044416/2da83aca-38c3-44ff-9a49-e3d676fac64b)

Long story short when I check PJ-1103 clamp power meter yaml file none of the id's match the iot,tuya devcie properties. and its pretty much the same with other proposed devices.

What can be the issue?
here are the screenshots of integration for the same device:
![image](https://github.com/make-all/tuya-local/assets/29044416/a9dc07ba-d299-4180-9b0d-3f244f056950)

![image](https://github.com/make-all/tuya-local/assets/29044416/79116bb8-88b2-41e4-8d78-64006f3e0fe2)

**Comment by make-all** [comment]
There is now a specific config for Atorch S1WP.
